pkg/util/util_test.go

```diff
@@ -0,0 +1,50 @@
+package util
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestE2E(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Util test")
+}
```

Comment:
Ginkgo is currently used only in e2e tests; unit tests are currently written using the stdlib only.

Can we do the same with this unit test to keep the consistency?

Also, the name of this function seems incorrect.

Reply:
Im wondering what is intention of using Ginkgo for e2e test, but stdlib for unit test?  After check more, I saw kubernetes/kubernetes code base has the same pattern.  Why not use one framework for both?

Reply:
Because ginkgo is considered more suitable for behaviour driven tests where it is easy to perform check from outside. unit tests etc use stdlib because unit tests aren't necessarily behaviour of k8s. There are some other reasons such as how tagging etc works for ginkgo tests.  For unit tests it is easy to focus and run just one file.

Reply:
K. updated test to use stdlib for unit testing.